### ISSUE TITLE
Pod Update is crashing during Generating Pods project.

### ISSUE THREAD

**Issue by thisisjak** [CocoaPods/CocoaPods#4878]
$ pod update
Update all pods
Updating local specs repositories

CocoaPods 1.0.0.beta.3 is available.
To update use: `sudo gem install cocoapods --pre`
[!] This is a test version we'd love you to try.

For more information see http://blog.cocoapods.org
and the CHANGELOG for this version http://git.io/BaH8pQ.

Analyzing dependencies
Fetching podspec for `NativeSDK` from `~/Documents/Developer/XXXXX/XXXXXXXXX`
Downloading dependencies
Using FXNotifications (1.1)
Using HockeySDK (3.6.4)
Installing MRProgress 0.8.3 (was 0.8.1)
Installing MTBBarcodeScanner 1.9.0 (was 1.4.0)
Using NativeSDK (3.0.5-SP05)
Generating Pods project

### Command

```
/Users/XXXXX/Software/ruby/bin/pod update
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?

### Stack

```
   CocoaPods : 0.39.0
        Ruby : ruby 2.0.0p645 (2015-04-13 revision 50299) [universal.x86_64-darwin15]
    RubyGems : 2.4.6
        Host : Mac OS X 10.11.4 (15E27e)
       Xcode : 7.2.1 (7C1002)
         Git : git version 2.5.4 (Apple Git-61)
Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 1dcacda0ad187a0421b1cc0061d5b7397c94988c
```

### Plugins

```
cocoapods-plugins : 0.4.2
cocoapods-search  : 0.1.0
cocoapods-stats   : 0.6.2
cocoapods-trunk   : 0.6.4
cocoapods-try     : 0.5.1
```

### Podfile

```ruby
target "SAPTSM" do
inhibit_all_warnings!

pod 'NativeSDK', :path => "~/Documents/XXXXXXXXXXX/SAP/NativeSDK"
pod 'FXNotifications', '~> 1.1'
pod 'MTBBarcodeScanner'
pod 'MRProgress'
pod 'HockeySDK', '~> 3.6.4'
end

target "SAPTSMTests" do

end
```

### Error

```
NoMethodError - undefined method `[]' for #<Xcodeproj::Project::Object::PBXFileReference:0x007f91d4c2e710>
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/project.rb:191:in `block in add_file_reference'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:266:in `block in each_filename'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:266:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/pathname.rb:266:in `each_filename'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/project.rb:189:in `add_file_reference'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer/file_references_installer.rb:184:in `block (2 levels) in add_file_accessors_paths_to_pods_group'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer/file_references_installer.rb:182:in `each'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer/file_references_installer.rb:182:in `block in add_file_accessors_paths_to_pods_group'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer/file_references_installer.rb:178:in `each'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer/file_references_installer.rb:178:in `add_file_accessors_paths_to_pods_group'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer/file_references_installer.rb:105:in `block in add_resources'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:140:in `message'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer/file_references_installer.rb:104:in `add_resources'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer/file_references_installer.rb:40:in `install!'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:602:in `install_file_references'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:155:in `block in generate_pods_project'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/user_interface.rb:59:in `section'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:153:in `generate_pods_project'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/installer.rb:111:in `install!'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:71:in `run_install_with_update'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/command/project.rb:156:in `run'
/Library/Ruby/Gems/2.0.0/gems/claide-0.9.1/lib/claide/command.rb:312:in `run'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/lib/cocoapods/command.rb:47:in `run'
/XXXXXXXXXXXXXX//ruby/gems/cocoapods-0.39.0/bin/pod:44:in `<top (required)>'
/XXXXXXXXXXXXXX//ruby/bin/pod:23:in `load'
/XXXXXXXXXXXXXX//ruby/bin/pod:23:in `<main>'
```

**Comment by segiddins** [comment]
This ought to be fixed in the latest 1.0 beta

**Comment by thisisjak** [comment]
Thanks Samuel!
